challenge_14/java/jdfurlan/src/ReversingALinkedList.java

```diff
@@ -0,0 +1,71 @@
+
+import java.util.Scanner;
+
+public class ReversingALinkedList {
+
+	private static void printList(ListNode head) {
+		while (head != null) {
+			System.out.print(head.data);
```

Comment:
This won't print a separator between node values so they will run together. Hopefully all your values end with a space.  ;)